### PR TITLE
Fix typos: 'lastest' -> 'latest'

### PR DESCRIPTION
**Pull request by zanglang**

### CODE REVIEW

**Review by lezzokafka — APPROVED** [review]
Thanks @zanglang :D

File: docs/getting-started/croeseid-testnet.md [hunk 1]
```diff
@@ -132,8 +132,8 @@ Follow the below optional steps to enable state-sync.
 - For state-sync configuration, in `~/.chain-maind/config/config.toml`, please modify the configurations of [statesync] `enable`, `rpc_servers`, `trust_height` and `trust_hash` by:
 
   ```bash
-  $ LASTEST_HEIGHT=$(curl -s https://testnet-croeseid.crypto.org:26657/block | jq -r .result.block.header.height); \
-  BLOCK_HEIGHT=$((LASTEST_HEIGHT - 1000)); \
+  $ LATEST_HEIGHT=$(curl -s https://testnet-croeseid.crypto.org:26657/block | jq -r .result.block.header.height); \
```
Comment (lezzokafka) [inline docs/getting-started/croeseid-testnet.md:135]:
```suggestion
  $ LATEST_HEIGHT=$(curl -s  https://testnet-croeseid-3.crypto.org:26657/block  | jq -r .result.block.header.height); \
```

Comment (lezzokafka) [reply docs/getting-started/croeseid-testnet.md:135]: @zanglang Oh I found that it should be connecting to the latest public fullnode of  `testnet-croeseid-3` which is https://testnet-croeseid-3.crypto.org:26657/block

File: docs/getting-started/croeseid-testnet.md [hunk 2]
```diff
@@ -132,8 +132,8 @@ Follow the below optional steps to enable state-sync.
 - For state-sync configuration, in `~/.chain-maind/config/config.toml`, please modify the configurations of [statesync] `enable`, `rpc_servers`, `trust_height` and `trust_hash` by:
 
   ```bash
-  $ LASTEST_HEIGHT=$(curl -s https://testnet-croeseid.crypto.org:26657/block | jq -r .result.block.header.height); \
-  BLOCK_HEIGHT=$((LASTEST_HEIGHT - 1000)); \
+  $ LATEST_HEIGHT=$(curl -s https://testnet-croeseid-3.crypto.org:26657/block | jq -r .result.block.header.height); \
+  BLOCK_HEIGHT=$((LATEST_HEIGHT - 1000)); \
   TRUST_HASH=$(curl -s "https://testnet-croeseid.crypto.org:26657/block?height=$BLOCK_HEIGHT" | jq -r .result.block_id.hash)
```
Comment (lezzokafka) [inline docs/getting-started/croeseid-testnet.md:137]:
```suggestion
  TRUST_HASH=$(curl -s "https://testnet-croeseid-3.crypto.org:26657/block?height=$BLOCK_HEIGHT" | jq -r .result.block_id.hash)
```

**Review by tomtau — APPROVED** [review]

**Pull request merged**